src/metabase/driver/sql/query_processor.clj

```diff
@@ -50,6 +50,7 @@
   :hierarchy #'driver/hierarchy)
 
 (defmethod current-datetime-fn :sql [_] :%now)
```

Comment:
Wouldn't it make more sense for this to be part of the `driver` protocol?

Reply:
Only `:sql` drivers need to supply this info because the only place it is used is in the SQL query processor. The main driver code has no use for that sort of info. It is only of use in generating queries. 

It would be pointless for MongoDB for example to have to implement this method and then call `(current-datetime-fn :mongo)` to get its own implementation

Reply:
at any rate `current-datetime-fn` is supposed to return a HoneySQL function name keyword (e.g. `:%function-name` which hardly makes sense for drivers that don't use SQL